Source/UZKFileInfo.m

```diff
@@ -64,7 +111,6 @@ - (NSDate *)timestamp {
 }
 
 
-
```

Comment:
Please remove any whitespace-only changes.

Reply:
This place only follows the blank lines reserved by your code style and has not been modified.

Reply:
In the upstream version of the file, there were three blank lines (65, 66, 67). The new version only has 2 (112, 113).

Reply:
Oh, I have always felt that your style is 2 lines above, and 2 lines below.

Reply:
**Does this belong to your style? I seem to be confused.**

![image](https://user-images.githubusercontent.com/5416852/61879314-5ff42f80-af25-11e9-8b21-585d69b32d81.png)

Reply:
It looks like there may be some places where I've lapsed. As more people contribute, it would be good for me to take a pass at cleaning everything up, making it consistent, and documenting the style guidelines. For now though, stick to whatever you think for new code, and leave existing code the way it is.

Reply:
OK, I will not submit new code for the time being until your unit test is completed, and we will continue to work to preserve your code style.

Reply:
You don't need to wait for me. I'm fine syncing in your changes as you make them.